scripts/run_az.py

```diff
@@ -0,0 +1,10 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import sys
+
+from azure.cli import __main__ as cli_main
+
+sys.exit(cli_main(sys.argv))
```

Comment:
Why this as opposed to debugging from: `${workspaceRoot}/src/azure-cli/azure/cli/__main__.py`?

Reply:
They may not have the CLI source cloned on their system.

Reply:
Aah, makes sense. Sounds good.